test/unit/app/common/state/tabStateTest.js

```diff
@@ -483,6 +483,10 @@ describe('tabState unit tests', function () {
       const expectedAppState = defaultAppState.set('tabs', tabsWithoutField)
       assert.deepEqual(newAppState, expectedAppState)
     })
+    it('returns the state (unchanged) if tabs is falsey', function () {
+      const newAppState = tabState.removeTabField(defaultAppState, 'loginRequiredDetail')
```

Comment:
yup! it gets passed as input- the `tabState` handle is just the reference to what got exported in `app/common/state/tabState.js`